python/GafferUI/PathPlugValueWidget.py

```diff
@@ -148,6 +149,7 @@ def _updateFromValues( self, values, exception ) :
 
 		self.pathWidget().setErrored( exception is not None )
 		self.__path.setFromString( sole( values ) or "" )
+		self.pathWidget().setPlaceholderText( self.__placeholderText() )
```

Comment:
Because PathPlugValueWidget doesn't support multiple plugs like StringPlugValueWidget does, it doesn't need to use `---` as a placeholder when it finds mixed values. This means it can do the placeholder management in an override for `_updateFromMetadata()`, which is called far less frequently than `_updateFromValues()`.